### ISSUE TITLE
Family Iron Still Not Set by Shovel?

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLifeData7#1222]
So when this was first address and updated, I had thought it was working until today.

What happened was this: 

- eved a family and returned them to an old town
- birthed as a male and placed stones a well site in the white band
- muddy iron became loose
- stripped one vein but then decided I wanted to keep looking
- found another well site in the ginger band
- placed the stones
- the muddy iron vein did become loose iron

I had expected, since I had not used the shovel yet, at the first well site, that this would not be the case?
Perhaps because I stripped one of the loose veins?

- returned to the first well site, removed all of the stones
- returned to the ginger site, placed stones
- the iron still didn't loosen
- went down to a white band well site
- placed the stones
- iron still didn't loosen
- returned to the original well site
- stripped veins, shoveled well
- all of the mines opened completely

The point of the update had been to prevent the family iron from being locked in until the family had at least tech'd up to a shovel.
This makes sense because it better predicts where the base is being developed.

Before, eve families could end up getting split or place stones on a bad spot (possibly with no iron), or the iron site could be lost.

**Comment by jasonrohrer** [comment]
No, it's working as intended.  The loose vein just gives you enough for your first shovel, but it also sets your primary homeland permanently.

Otherwise, you could run around getting loose iron everywhere by building well sites.
